Tear down the Friend of Jenkins program, document how to get recognition for donations (#4343)
jenkins-infra/jenkins.io: content/donate.adoc

== Donations through Continuous Delivery Foundation 

Large donations can be also made through link:https://cd.foundation/[Continuous Delivery Foundation].
Please contact the link:/project/board/[Jenkins Governance Board] if you want to make such a donation.

== Recognition

As for any other type of contributions, we are interested in publicly recognizing and promoting donations made to the project.
If you announce your or your company's donation in social media, reach out to the link:/sigs/advocacy-and-outreach/[Advocacy and Outreach special interest group] for reposts.
The Jenkins governance board is also responsible for recognizing big donations.
There is no formal process for that, please contact the link:/project/board/[Jenkins Governance Board].

== Friend of Jenkins

> The Jenkins project used to send a special "Friend of Jenkins" plugin to contributors who donate 25 USD or more.

Author: I decided to keep it just in case there are useful anchors pointing to that page.
Once/if we have the Friend of Jenkins plugin open sourced, I will replace the text to a recommendation to install this plugin after the donation.

> The Jenkins project used to send a special "Friend of Jenkins" plugin to contributors who donate 25 USD or more.

Reviewer: The "used to" is kinda weak, I prefer something a bit more explicit to make it clear that it was done in the past.

```suggestion
Before May 15th 2021, the Jenkins project would send a special "Friend of Jenkins" plugin to contributors who donate 25 USD or more. This setup not only went against our design of an open source project, but it didn't adequately acknowledged the thanks we wanted to give people. We are working on redefining this program. Please refer to link:https://groups.google.com/u/1/g/jenkinsci-dev/c/bIgDEM2E7hY[this thread] for the details.
```

> The Jenkins project used to send a special "Friend of Jenkins" plugin to contributors who donate 25 USD or more.

Author: Do not disagree about the original message being weak. Let's do it!

> Before May 15th 2021, the Jenkins project would send a special "Friend of Jenkins" plugin to contributors who donate 25 USD or more. This setup not only went against our design of an open source project, but it didn't adequately acknowledged the thanks we wanted to give people. We are working on redefining this program. Please refer to link:https://groups.google.com/u/1/g/jenkinsci-dev/c/bIgDEM2E7hY[this thread] for the details.

Reviewer 2: ```suggestion
Before May 15th 2021, the Jenkins project would send a special "Friend of Jenkins" plugin to contributors who donate 25 USD or more. This setup not only went against our design of an open source project, but it didn't adequately acknowledge the thanks we wanted to give people. We are working on redefining this program. Please refer to link:https://groups.google.com/u/1/g/jenkinsci-dev/c/bIgDEM2E7hY[this thread] for the details.
```

> This program is no longer active, and we are in process of redefining it.

Reviewer: ```suggestion
```

This got doubled up

> Please refer to link:https://groups.google.com/u/1/g/jenkinsci-dev/c/bIgDEM2E7hY[this thread] for the current state.

Reviewer: ```suggestion
```
